pkg/controllers/unifiedauth/unified_auth_controller.go

```diff
@@ -256,8 +261,9 @@ func (c *Controller) newClusterRoleBindingMapFunc() handler.MapFunc {
 }
 
 // found out which clusters need to sync impersonation config from rules like:
-//   resources: ["cluster/proxy"]
-//   resourceNames: ["cluster1", "cluster2"]
+//
```

Comment:
Why add the blank line here?
Since we are going to cherry-pick this to release branches, we should keep this patch minimal to avoid conflict.

Reply:
Sorry, vscode auto formatted the file. I reverted the change.